Readme.md

```diff
@@ -0,0 +1,46 @@
+# Setup Steps
+
+- If using docker see [Docker](#docker)
+
+- If python virtual environment is already created for your local docs directory see [Usage](#usage) steps else start with [Installation](#installation).
+
+- Checkout the docs directory and execute the below commands.
```

Comment:
"Checkout this repo and execute the below commands."

Reply:
resolved in pr #4